.eslintrc.js

```diff
@@ -10,10 +10,16 @@ module.exports = {
     'prettier/prettier',
     'plugin:prettier/recommended',
   ],
-
-  env: {
-    node: true,
-  },
-
+  env: {node: true},
   ignorePatterns: ['dist'],
+  plugins: ['simple-import-sort'],
+  rules: {
+    'simple-import-sort/imports': 'error',
+    'simple-import-sort/exports': 'error',
+    'import/first': 'error',
+    'import/newline-after-import': 'error',
+    // no-duplicate-imports does't have autofix
+    'no-duplicate-imports': 'off',
+    'import/no-duplicates': 'error',
+  },
```

Comment:
✨ NIT: IMHO the prettier plugin is a bit faster: [`prettier-plugin-organize-imports`](https://www.npmjs.com/package/prettier-plugin-organize-imports)

Reply:
I've done a quick search today and indeed noticed there are prettier plugins like this that I wasn't aware of. however, I'll stay with my classic, self-confirming reasons counting: it's more popular, I'm used to it, prettier runs as eslint plugin anyway and this will have its own rule, not cryptic `prettier/prettier`... though I would be interested in seeing performance comparison